piqe_ocp_lib/tests/resources/test_ocp_machine_management.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import pytest
+from pprint import pprint
```

Comment:
It doesn't look like pprint is being used anywhere.

Reply:
Will fix